python/whylogs/experimental/core/metrics/udf_metric.py

```diff
@@ -236,12 +238,11 @@ def decorator_register(func):
     return decorator_register
 
 
-def generate_udf_schema() -> List[ResolverSpec]:
+def generate_udf_metric_schema(non_udf_resolvers: List[ResolverSpec] = STANDARD_RESOLVER) -> DeclarativeSchema:
```

Comment:
The default value in the signature should be None and then set inside to avoid global instance of the list.